data/common.yaml

```diff
@@ -117,6 +128,17 @@ scriptworker_users:
         - notary8
         - notary9
         - notary10
+  vpn:
+    cltbld:
+      home: /builds/scriptworker
+      supported_behaviors:
+        - mac_notarize_vpn
+      notarization_users:  # Do we need this?
+        - notary1
```

Comment:
Really only needed if we want to start using https://github.com/mozilla-releng/scriptworker-scripts/blob/3fc292ffca72364787d2ddd7e7c6e1220b2d7f54/iscript/src/iscript/mac.py#L719 again.

We tend to get rate limited that way, so we found submitting one big zip is more reliable. We haven't needed sudo-wrapped notarization in a while, so we could potentially remove it from iscript.